install.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import logging
+from pathlib import Path
+
+try:
+    from helperFunctions.install import OperateInDirectory, is_virtualenv, run_cmd_with_logging, check_distribution
+    from plugins.installer import AbstractPluginInstaller
+except ImportError:
+    import sys
+    SRC_PATH = Path(__file__).absolute().parent.parent.parent.parent
+    sys.path.append(str(SRC_PATH))
+
+    from helperFunctions.install import OperateInDirectory, is_virtualenv, run_cmd_with_logging, check_distribution
+    from plugins.installer import AbstractPluginInstaller
+
+
+class CodescannerInstaller(AbstractPluginInstaller):
+    base_path = Path(__file__).resolve().parent
+
+    def build(self):
+        build_dir = self.base_path / 'build' / 'Codescanner'
+        run_cmd_with_logging(
+            f'git clone https://github.com/fkie-cad/Codescanner.git -o Codescanner', raise_error=True
+        )
+        with OperateInDirectory(build_dir, remove=True):
+            command = 'python3 setup.py install'
+            if not is_virtualenv():
+                logging.warning("no venv")
+                command = f'sudo -EH {command}'
```

Comment:
This does not work for non-virtual installations.

The install succeeds, but the package is only accessible by the root user. Running fact as non-root yields:

```python3
**  File "/home/fact/FACT_core/src/./plugins/analysis/codescanner_plugin/routes/routes.py", line 3, in <module>
    from codescanner_analysis import CodescannerAnalysisData
ModuleNotFoundError: No module named 'codescanner_analysis**
```

And, indeed, if I try to import `codescanner_analysis` using the same user, it doesn't work:

```bash
$ python3 -c "import codescanner_analysis"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'codescanner_analysis'
$
```

However, using root everything is fine:

```bash
$ sudo python3 -c "import codescanner_analysis"
$
```

Reply:
Great, works flawlessly now